tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -532,6 +532,11 @@ public static class CIVisibility
             /// Configuration key for enabling or disabling the early flake detection feature in CI Visibility
             /// </summary>
             public const string EarlyFlakeDetectionEnabled = "DD_CIVISIBILITY_EARLY_FLAKE_DETECTION_ENABLED";
+
+            /// <summary>
+            /// Configuration key for set the rum flushing wait in milliseconds
+            /// </summary>
+            public const string RumFlushWaitMillis = "DD_CIVISIBILITY_RUM_FLUSH_WAIT_MILLIS";
```

Comment:
Don't forget to add this to the dd-go telemetry

Reply:
already there :)